core/src/test/java/nl/nn/adapterframework/jdbc/JdbcTransactionalStorageTest.java

```diff
@@ -0,0 +1,222 @@
+package nl.nn.adapterframework.jdbc;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
+
+import java.io.ByteArrayOutputStream;
+import java.io.ObjectOutputStream;
+import java.io.OutputStream;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Date;
+import java.util.zip.DeflaterOutputStream;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import liquibase.Liquibase;
+import liquibase.database.jvm.JdbcConnection;
+import liquibase.resource.FileSystemResourceAccessor;
+import nl.nn.adapterframework.stream.Message;
+import nl.nn.adapterframework.testutil.TestFileUtils;
+
+public class JdbcTransactionalStorageTest extends TransactionManagerTestBase {
+
+	private JdbcTransactionalStorage<Message> storage;
+	private Liquibase liquibase;
+	private boolean tableCreated = false;
+	private final String tableName = "IBISSTORE";
+	private final String messageField = "MESSAGE";
+	private final String keyField = "MESSAGEKEY";
+
+	public JdbcTransactionalStorageTest() {}
```

Comment:
Codacy says this constructor is not necessary, and is right about that!
Please also have a look at the other Codacy issues. It's always nice to get a green mark from Codacy